api/cloud/v1.md

```diff
@@ -0,0 +1,116 @@
+---
+title: CockroachDB Cloud API
+summary: Learn about the CockroachDB Cloud API
+layout: api
+docs_area: reference.api
+---
+
+{% assign releases = site.data.cloud_api_releases %}
+
+<style>
```

Comment:
Nit: Could we pull this CSS out into a separate file?

Reply:
Unfortunately no. I'm not clear why, but adding these CSS elements to our main CSS files ended up getting overriden by RapiDoc.